Add oidc flags for CAPI clusters (#679)
giantswarm/kubectl-gs: CHANGELOG.md

## [Unreleased]

### Added

> - Add OIDC flags on the template cluster command for Cluster API.

Reviewer: ```suggestion
- Add OIDC flags to the `template cluster` command (OpenStack only).
```
